ansible/files/spark/airflow.sh

```diff
@@ -0,0 +1,112 @@
+set -o verbose
+
+HOME=/home/hadoop
+source $HOME/.bashrc
+
+# Error message
+error_msg ()
+{
+    echo 1>&2 "Error: $1"
+}
+
+# Parse arguments
+while [ $# -gt 0 ]; do
+    case "$1" in
+        --job-name)
+            shift
+            job_name=$1
+            ;;
+        --user)
+            shift
+            user=$1
+            ;;
+        --uri)
+            shift
+            uri=$1
+            ;;
+        --arguments)
+            shift
+            args=$1
+            ;;
+        --data-bucket)
+            shift
+            data_bucket=$1
+            ;;
+        --environment)
+            shift
+            environment=$1
+            ;;
+         -*)
+            # do not exit out, just note failure
+            error_msg "unrecognized option: $1"
+            ;;
+          *)
+            break;
+            ;;
+    esac
+    shift
+done
+
+if [ -z "$job_name" ] || [ -z "$user" ] || [ -z "$uri" ] || [ -z "$data_bucket" ]; then
+    error_msg "missing argument(s)"
+    exit -1
+fi
+
+s3_base="s3://$data_bucket/data/$user/$job_name"
```

Comment:
I'm OK with this. Currently the 'run a job' and 'publish data' tasks are one and the same. It would be a good idea to separate them and take a more deliberate approach to publishing data for use by dashboards etc.